tests/robot-tests/tests/admin_and_public/bau/publish_data.robot

```diff
@@ -528,18 +530,18 @@ Select table featured table from subjects step
     user clicks link    Test highlight name
     user waits until results table appears    %{WAIT_LONG}
     user waits until page contains element
-    ...    xpath://*[@data-testid="dataTableCaption" and text()="Admission Numbers for '${SUBJECT_2_NAME}' for Not specified in Bolton 001 (E02000984), Bolton 001 (E05000364), Bolton 004 (E02000987), Bolton 004 (E05010450), Nailsea Youngwood and Syon between 2005 and 2020"]
+    ...    xpath://*[@data-testid="dataTableCaption" and text()="Admission Numbers for '${SUBJECT_2_NAME}' for Not specified in Bolton 001, Bolton 004, Nailsea Youngwood and Syon between 2005 and 2017"]
 
 Validate table column headings for featured table
     user checks table column heading contains    1    1    Admission Numbers
 
 Validate table rows for featured table
-    ${row}=    user gets row number with heading    Bolton 001 (E02000984)
-    user checks table heading in offset row contains    ${row}    0    2    2019
+    ${row}=    user gets row number with heading    Bolton 001
+    user checks table heading in offset row contains    ${row}    0    2    2009
 
-    user checks table cell in offset row contains    ${row}    0    1    8,533
+    user checks table cell in offset row contains    ${row}    0    1    5,815
 
-    ${row}=    user gets row number with heading    Bolton 001 (E05000364)
+    ${row}=    user gets row number with heading    Bolton 001
```

Comment:
We're calling `${row}= user gets row number with heading Bolton 001` twice here.

Are there two rows labelled "Bolton 001" in the table? Might make it tough to get the row you want - not sure if we provide a way to grab a particular row by the row number instead.